esmvaltool/recipes/ref/recipe_model_benchmarking_timeseries_region.yml

```diff
@@ -0,0 +1,295 @@
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    area_statistics:
+      operator: mean
+
+  pp:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    anomalies:
+      period: month
+      reference:
+        start_year: 1980
+        start_month: 1
+        start_day: 1
+        end_year: 2009
+        end_month: 12
+        end_day: 31
+    area_statistics:
+      operator: mean
+
+  pp_200:
+    extract_levels:
+      levels: 20000
+      scheme: linear
+      coordinate: air_pressure
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    anomalies:
+      period: month
+      reference:
+        start_year: 1980
+        start_month: 1
+        start_day: 1
+        end_year: 2009
+        end_month: 12
+        end_day: 31
+    area_statistics:
+      operator: mean
+
+
+diagnostics:
+
+  timeseries:
+    description: Plot "classical" time series of global mean anomalies including a reference dataset.
+    variables:
+      tas: &var_default
+        start_year: 1980
+        end_year: 2014
+        preprocessor: pp
+        project: CMIP6
+        mip: Amon
+        exp: historical
+        ensemble: r1i1p1f1
+        grid: gr
+        reference_dataset: HadCRUT5
+        additional_datasets:
+          - {dataset: HadCRUT5, project: OBS, type: ground,
+             version: 5.0.1.0-analysis, tier: 2,
+             reference_for_metric: true}
+      pr:
+        <<: *var_default
+        reference_dataset: GPCP-SG
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, version: 2.3, tier: 2,
+             reference_for_metric: true}
+      psl:
+        <<: *var_default
+        reference_dataset: ERA5
+        additional_datasets:
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true}
+      hus_200:
+        <<: *var_default
+        short_name: hus
+        reference_dataset: ERA5
+        preprocessor: pp_200
+        additional_datasets:
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true}
+      ua_200:
+        <<: *var_default
+        short_name: ua
+        reference_dataset: ERA5
+        preprocessor: pp_200
+        additional_datasets:
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true}
+    scripts:
+      allplots:
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        group_variables_by: variable_group
+        plots:
+          timeseries:
+            annual_mean_kwargs: False
+            plot_kwargs:
+              MIROC6:
```

Comment:
Again here, it would be great if we could have the model lines as "default".

Reply:
Thanks, changed